fastlane/lib/fastlane/actions/device_grid/AppetizeFastfile

```diff
@@ -0,0 +1,22 @@
+desc "Generate and upload an .app to appetize.io"
+desc "Import this lane and call it from Danger"
+
+private_lane :build_and_upload_appetize do |options|
+  tmp_path = "/tmp/fl_build"
+
+  xcodebuild_configs = options[:xcodebuild]
+  xcodebuild_configs[:sdk] = "iphonesimulator"
+  xcodebuild_configs[:derivedDataPath] = tmp_path
+  
+  xcodebuild(xcodebuild_configs)
+
+  app_path = Dir[File.join(tmp_path, "**", "*.app")].last
```

Comment:
Good point, I think we have a class for that in FastlaneCore somewhere, I'll take a look